CMakeLists.txt

```diff
@@ -573,12 +573,12 @@ endif()
 if( CMAKE_SYSTEM_NAME MATCHES "Darwin|Windows" )
    set( APP_NAME "Saucedacity" )
 else()
-   set( APP_NAME "saucedacity" )
+   set( APP_NAME "tenacity" )
    set( PROPER_NAME "Saucedacity" )
 endif()
 
 # Define Application ID
-set( APP_ID "io.github.saucedacity.Saucedacity" )
+set( APP_ID "io.github.tenacity.Saucedacity" )
```

Comment:
Careful there!

Reply:
Whoops! That's not correct. It should be org.tenacityaudio.Tenacity instead.

I forgot to mention that I ran 'sed -i "s/saucedacity/tenacity/g" *' on the codebase several times, even using `find`. This is not deliberate and will be changed appropriately.